apps/api/src/router/client.routes.js

```diff
@@ -4,16 +4,28 @@ import { middlewares } from "../middleware/auth";
 
 const router = Router();
 
-router.get("/", middlewares.ensureAuthenticated, ClientController.getClients);
-router.get("/:id", middlewares.ensureAuthenticated, ClientController.getById);
+router.get(
+  "/",
+  middlewares.ensureAuthenticated,
+  middlewares.hasRole("ViewClients"),
```

Comment:
We might support multiple roles check, probably one endpoint in the future or later might access multiple resources not only client resources related 🤔 

Also it'd be better to move the values `"ViewClients"` to a constants file
something like this

```suggestion
  middlewares.hasRole([PERMISSIONS.VIEW_CLIENTS]),
```